etc/schema.json

```diff
@@ -1156,7 +1156,12 @@
                         "opcode": {
                             "description": "DNS opcode as an integer",
                             "type": "integer"
-                        }
+                        },
+			"authorities": {
+			    "description": "Array of authorities in DNS response",
+			    "$comment": "Probably also need answers and additional records",
+			    "type": "array"
```

Comment:
I think this should be completed by the list of items and `"additionalProperties": false`

I think we do not want quick and dirty green CI

Reply:
My purpose is to unblock some work before getting to the proper fixes.